catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/DuplicateCommands.java

```diff
@@ -253,6 +285,13 @@ protected long getFilterStartTime(long now) {
         return startTime;
     }
 
+    protected String getTemporalProperty() {
+        if (StringUtils.isNotEmpty(temporalProperty) && isValidTemporalProperty(temporalProperty)) {
+            return temporalProperty;
```

Comment:
So SOLR fields are case-insensitive then?  I am not sure if they are, but if this works I am guessing they are.